### DISCUSSION TITLE
Web-ready Image Format that Supports Lab Color?

### DISCUSSION THREAD

**Discussion by blue-j** [ImageMagick/ImageMagick#3447] [General]
Is there such a thing?  Anyone know of a web format that supports Lab and will be displayed by a browser?  - J

**Comment by snibgo** [comment]
No such thing, as far as I know.

TIFF is a good format for storing Lab images, but web browsers can't usually read TIFF. At least, Firefox can't, but perhaps some other browsers can these days.

**Reply by blue-j** [reply]
You can have a JP2 in Lab, but Chrome spits it out... I'll test with other browsers... It's too bad, if true!
